platform/darwin/src/MGLStyleLayer.h

```diff
@@ -63,7 +63,7 @@ MGL_EXPORT
 @property (nonatomic, assign, getter=isVisible) BOOL visible;
 
 /**
- The maximum zoom level at which the layer gets parsed and appears.
+ The maximum zoom level that the layer gets parsed and appears up to (but not including).
```

Comment:
The multiple dangling prepositions are a bit awkward here. Also, I missed this before, but “parsing” a layer only makes sense in the context of style JSON, not runtime styling. How about:

> The exclusive upper bound of the range of zoom levels within which a map would display the layer.

For consistency, the `minimumZoomLevel` description would become:

> The inclusive lower bound of the range of zoom levels within which a map would display the layer.